etc/kayobe/ipa.yml

```diff
@@ -46,7 +46,12 @@ ipa_build_dib_elements_extra:
 
 # Dictionary of additional environment variables to provide to Diskimage
 # Builder (DIB) during IPA image build.
-#ipa_build_dib_env_extra:
+ipa_build_dib_env_extra:
+  # This is to workaround the fact that pip > 10 will produce an error if you try and
+  # uninstall a distuils installed package. Previous versions would remove the
+  # metadata only -  leaving the code intact, see: https://bugs.launchpad.net/diskimage-builder/+bug/1768135
+  DIB_INSTALLTYPE_pip_and_virtualenv: package
```

Comment:
wow, the mix of caps and lower case for the same variable is slightly startling.